caim-org/caim-app
# [Caim-58] Increase the email field character limit

## Purpose

Extend all email addresses max length to 255 charecters.

## Changes 

- changes email attributes to 255 max_length for models
- adds migration file 

## QA/ Notes 

Remember because we are directly modifiy the user model, so it's migrations still needs to be generated in django outside of this PR, so remember to run `makemigrations` to change that, before running `migrate` on all these together.  That's a confusing way to put it but you should see:
```
Migrations for 'auth':
  caim-app/lib/python3.9/site-packages/django/contrib/auth/migrations/0014_alter_user_email.py
    - Alter field email on user
```
even though that migration is not included in this PR or important to git.

## Ticket

https://caimcommunity.atlassian.net/browse/CAIM-58
